### ISSUE TITLE
No execute permissions on /usr/share/backuppc/bin/BackupPC

### ISSUE THREAD

**Issue by acranox** [raoulbhatia/backuppc#16]
Building a package of 4.3.2 from the latest commit, 4e32d29e2b21f8282128b9fb2530fb1ed1ceb25f I ended up with it installing the program without execute permissions, so the service failed to start.
I'm not sure which commit introduced this behavior, but building from b1bbdd5148225237f963fe32058cd9e2c75feaf6 worked okay.

```
$ sudo bash -x  /etc/init.d/backuppc status
+ set -e
+ BINDIR=/usr/share/backuppc/bin
+ DATADIR=/var/lib/backuppc
+ USER=backuppc
+ NICE=0
+ test -f /etc/default/backuppc
+ . /etc/default/backuppc
++ NICE=0
+ NAME=backuppc
+ DAEMON=BackupPC
+ test -x /usr/share/backuppc/bin/BackupPC
+ exit 0


$ ls -l /usr/share/backuppc/bin/BackupPC
-rw-r--r-- 1 root root 87187 Jun 21 03:31 /usr/share/backuppc/bin/BackupPC
```

**Comment by raoulbhatia** [comment]
Closed via 0c7255f4cdaf5e5bc796a2c324675ed75e9f04e6

**Comment by raoulbhatia** [comment]
Enhancement in 8cbeb6d1711717b44b4f4f842e08a36eaff00e84

**Comment by raoulbhatia** [comment]
Hopefully final fix via 9c4f51a60e7de6395dd8ac4111f899f6f28120e5 ;-)
